index.bs

```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
+Group: w3c/webextensions
+URL: https://github.com/w3c/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla, https://www.mozilla.org/, tjovanovic@mozilla.com

```

Comment:
These two Editor lines use different syntax; have you tried this in Bikeshed and do they produce the same result?

Reply:
Indeed there was a difference (because of one comma). Thanks